package.json

```
@@ -9,7 +9,7 @@
     "copy:all": "rsync -a --include '*/' --include '*.js' --exclude '*' $(pwd)/specs/ ~/.fig/autocomplete/",
     "create-boilerplate": "./scripts/create-boilerplate.sh",
     "create-example": "./scripts/create-example.sh",
-    "build": "ts-node-script scripts/compiler.ts",
+    "build": "esbuild dev/*.ts dev/**/*.ts --outdir=specs --minify --format=esm --bundle",
```

Not sure about this but I think this will only work if your shell expands glob paths. It may be interesting to run `compiler.ts` instead, and get the TS files to build from there.

The compiler will also need an option `watch` to watch or not the files. This would also avoid duplicating build options (outdir, minify, format, etc)

---

We may create a new `watch` npm script and also keep the static `build` one.
```suggestion
    "build": "esbuild dev/*.ts dev/**/*.ts --outdir=specs --minify --format=esm --bundle",
    "watch": "ts-node-script scripts/compiler.ts",
```

---

> We may create a new `watch` npm script and also keep the static `build` one.

There is already the `dev` script that does this

---

Oops, sorry😆